app/src/main/java/co/nimblehq/compose/crypto/ui/screens/price/PriceChange.kt

```diff
@@ -22,7 +24,8 @@ import kotlin.math.abs
 fun PriceChange(
     priceChangePercentage24hInCurrency: Double,
     modifier: Modifier,
-    iconPaddingEnd: Dp
+    iconPaddingEnd: Dp,
+    textStyle: TextStyle = Style.medium16()
```

Comment:
If we modify the Composable like this for reusing purposes, It will be coupled and difficult to update in the future. I'd recommend creating a new Composable for the price change in detail page. 

Or we could have a flag `displayForDetailPage = true/false` rather than passing the padding and textStyle in the params. What do you think? 😄

Reply:
> have a flag displayForDetailPage = true/false rather than passing the padding and textStyle in the params.

Good idea @hoangnguyen92dn I will update 🚀